packages/application/src/application/services/pull-request.types.ts

```diff
@@ -54,10 +58,44 @@ export function mapApplicationToPullRequestFile(application: DatacapAllocator):
     }
   }
 
+ //get current values of msig
+  let allocatorId     = application.allocatorMultisigAddress ?? ""
+  let updatedSigners  = application.allocatorMultisigSigners ?? []
+
+  //if we have an on-chain address, fetch the latest from Filfox
+  if (application.allocatorMultisigAddress) {
+    try {
+      const msigData = await getMultisigInfo(
+        application.allocatorMultisigAddress
+      )
+
+      if (msigData.id !== application.allocatorMultisigAddress) {
+        allocatorId = msigData.id
+      }
+
+      const fetchedSigners = msigData.multisig.signers
+      if (
+        JSON.stringify(fetchedSigners) !==
+        JSON.stringify(application.allocatorMultisigSigners)
+      ) {
+        updatedSigners = fetchedSigners
+      }
+    } catch (err) {
+      console.error(
+        `mapApplicationToPullRequestFile: failed to fetch Filfox info for ${
+          application.allocatorMultisigAddress
+        }:`,
+        err
+      )
+      // keep the old values if the fetch fails
```

Comment:
```suggestion
      /* Note to future maintainers: if *any part* of this fails,
        we must keep the old values otherwise things can get
        wacky. If you introduce code later which can cause
        exceptions from anywhere other than `getMultisigInfo`
        then make sure you maintain this promise!
```